File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.cpp
class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        vector<int> sumVec;
        for(int i=0; i<nums.size(); i++){
            int sum = nums[i];
            for(int j=0; j<i; j++){
                sum += nums[j];
            }
            sumVec.push_back(sum);
            
        }
        return sumVec;
    }
};

File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.py
class Solution(object):
    def runningSum(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        ret = []
        for i in range(len(nums)):
            if(i==0):
                ret.append(nums[0])
            else:
                last = ret.pop()
                ret.append(last)
                ret.append(last+nums[i])

        return ret